src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/StacksScenario.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.UnitTestApp.Scenarios
+{
+    internal class StacksScenario
+    {
+        [DllImport("MonitorProfiler", CallingConvention = CallingConvention.StdCall, PreserveSig = true)]
```

Comment:
Likely needs to be libMonitorProfiler for Linux builds.

Reply:
That's not necessary to do. The runtime seems to be searching for the platform variations of the name, but it's searching in the wrong path:

```
[App1] Unhandled exception. System.DllNotFoundException: Unable to load shared library 'MonitorProfiler' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: 
[App1] dlopen(/Users/runner/work/1/s/.dotnet/shared/Microsoft.NETCore.App/7.0.0-rc.2.22430.11/MonitorProfiler.dylib, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/artifacts/bin/Microsoft.Diagnostics.Monitoring.UnitTestApp/Debug/net7.0/MonitorProfiler.dylib, 1): image not found
[App1] dlopen(MonitorProfiler.dylib, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/.dotnet/shared/Microsoft.NETCore.App/7.0.0-rc.2.22430.11/libMonitorProfiler.dylib, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/artifacts/bin/Microsoft.Diagnostics.Monitoring.UnitTestApp/Debug/net7.0/libMonitorProfiler.dylib, 1): image not found
[App1] dlopen(libMonitorProfiler.dylib, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/.dotnet/shared/Microsoft.NETCore.App/7.0.0-rc.2.22430.11/MonitorProfiler, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/artifacts/bin/Microsoft.Diagnostics.Monitoring.UnitTestApp/Debug/net7.0/MonitorProfiler, 1): image not found
[App1] dlopen(MonitorProfiler, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/.dotnet/shared/Microsoft.NETCore.App/7.0.0-rc.2.22430.11/libMonitorProfiler, 1): image not found
[App1] dlopen(/Users/runner/work/1/s/artifacts/bin/Microsoft.Diagnostics.Monitoring.UnitTestApp/Debug/net7.0/libMonitorProfiler, 1): image not found
[App1] dlopen(libMonitorProfiler, 1): image not found
```